BitwardenShared/UI/Auth/TwoFactorNotice/SetUpTwoFactor/SetUpTwoFactorView.swift

```diff
@@ -0,0 +1,114 @@
+import SwiftUI
+
+// MARK: - SetUpTwoFactorView
+
+/// A view that alerts the user to the new policy of sending emails to confirm new devices.
+///
+struct SetUpTwoFactorView: View {
+    // MARK: Properties
+
+    /// An object used to open urls from this view.
+    @Environment(\.openURL) private var openURL
+
+    /// An environment variable for getting the vertical size class of the view.
+    @Environment(\.verticalSizeClass) var verticalSizeClass
+
+    /// The `Store` for this view.
+    @ObservedObject public var store: Store<SetUpTwoFactorState, SetUpTwoFactorAction, SetUpTwoFactorEffect>
+
+    var body: some View {
+        VStack(spacing: 12) {
+            DynamicImageTextStackView(minHeight: 0) {
+                Asset.Images.Illustrations.userLock.swiftUIImage
+                    .resizable()
+                    .frame(
+                        width: verticalSizeClass == .regular ? 152 : 124,
+                        height: verticalSizeClass == .regular ? 152 : 124
+                    )
+                    .accessibilityHidden(true)
+            } textContent: {
+                VStack(spacing: 16) {
+                    Text(Localizations.setUpTwoStepLogin)
+                        .styleGuide(.title, weight: .bold)
+
+                    Text(Localizations.youCanSetUpTwoStepLoginAsAnAlternative)
+                        .styleGuide(.title3)
+                }
+                .padding(.horizontal, 12)
+            }
+
+            Button {
+                store.send(.turnOnTwoFactorTapped)
+            } label: {
+                Label {
+                    Text(Localizations.turnOnTwoStepLogin)
+                } icon: {
+                    Asset.Images.externalLink24.swiftUIImage
+                }
+            }
+            .buttonStyle(.primary())
+
+            Button {
+                store.send(.changeAccountEmailTapped)
+            } label: {
+                Label {
+                    Text(Localizations.changeAccountEmail)
+                } icon: {
+                    Asset.Images.externalLink24.swiftUIImage
+                }
+            }
+            .buttonStyle(.secondary())
+
+            if store.state.allowDelay {
+                AsyncButton(Localizations.remindMeLater) {
+                    await store.perform(.remindMeLaterTapped)
+                }
+                .buttonStyle(.secondary())
+            }
+
+            Spacer()
```

Comment:
🤔 Is this `Spacer()` necessary? It seems like this could be for pushing up the content, but the content in the scroll view should already be at the top? Or if it's just padding, I think the scroll view modifier also adds bottom padding.

Reply:
At the time I had it in there, I needed it to get things pinned to the top, but I also changed around how this view was constructed since then. I'll double-check.